agent/test_data/PRIMARY_PASSWORD

```diff
@@ -1 +1 @@
-test-password
\ No newline at end of file
+rootpassword
```

Comment:
It looks better to use different test passwords for different purposes.
Can we avoid the possibility to confuse the passwords in this directory?

Reply:
It's good. However, it needs a lot of changes.
So I will do it in another PR.

https://github.com/cybozu-go/moco/pull/147